nilearn/tests/test_ndimage.py

```diff
@@ -21,6 +24,43 @@ def test_largest_cc():
     b[5, 5, 5] = 1
     np.testing.assert_equal(a, largest_connected_component(b))
 
+    img = testing.generate_labeled_regions(shape = (10,11,12),
+                                           n_regions = 2 )
+
+    assert_raises(ValueError, largest_connected_component, img)
```

Comment:
Can you write a short comment here of what you are testing ?
It will be useful for anybody who can quickly follow and understand.

Same for both tests you have added here.